x/execution/internal/keeper/keeper.go

```diff
@@ -19,7 +19,10 @@ import (
 )
 
 // price share for the execution runner
-var runnerShare = sdk.NewDecWithPrec(9, 1)
+var (
+	runnerShare   = sdk.NewDecWithPrec(8, 1)
+	emittersShare = sdk.NewDecWithPrec(1, 1)
```

Comment:
just for comprehension, it would be nice to also have the `serviceShare` and assert in the code if the rest of `runner + emitters != serviceShare`. Like that, we will have 100%

Reply:
Yes and no. We need to be sure that inputs == outputs and we can only have that, if we subtract it, because division can't guarantee that.

Reply:
of course but we can still check that the diff matches the supposed value of the service. Anyway not blocking but it's just really confusing not to have all the repartition here, it looks like there is only 90% of the distribution and the rest maybe stays in the execution. (If we have a deep look at the code then we see it's not the case but we should not need to have a deep look at the code)